packages/typespec-vs/src/Microsoft.TypeSpec.VS.csproj

```diff
@@ -40,12 +47,18 @@
     </Content>
   </ItemGroup>
   <ItemGroup>
+    <!-- The 'Microsoft.VisualStudio.Composition' is not used directly but referenced by nuget package 'Microsoft.VisualStudio.Workspace' -->
+    <!-- Have to add it explicitly here because otherwise 'Microsoft.VisualStudio.Workspace" will report warning NU1603 because its desired version not found and higher version will be used -->
+    <!-- TODO: remove this explicit reference after 'Microsoft.VisualStudio...' releases new version without the warning-->
```

Comment:
Please file an issue for this and reference in the TODO

Reply:
#5368 filed to track the issue.